docs/rules/no-duplicate-imports.md

```diff
@@ -1,6 +1,8 @@
 # Disallow duplicate imports (no-duplicate-imports)
 
-An ES6/ES2015 import can be spread over multiple lines, but this takes up unneeded whitespace. This rules validates that all imports from a single module exists in a single import statement.
+An ES6/ES2015 import can be spread over multiple lines, but this takes up unneeded whitespace.
```

Comment:
Personally I think it's clearer if the sentence is moved back up (with your suggested phrasing)